modeling-cmds/src/shared.rs

```diff
@@ -880,6 +880,34 @@ pub enum ExtrusionFaceCapType {
 }
 impl_string_enum_sql! {ExtrusionFaceCapType}
 
+/// Post effect type
+#[allow(missing_docs)]
+#[derive(
+    Display,
+    FromStr,
+    Copy,
+    Eq,
+    PartialEq,
+    Debug,
+    JsonSchema,
+    Deserialize,
+    Serialize,
+    Sequence,
+    Clone,
+    Ord,
+    PartialOrd,
+    ExecutionPlanValue,
+)]
+#[cfg_attr(feature = "diesel", derive(AsExpression, FromSqlRow))]
+#[cfg_attr(feature = "diesel", diesel(sql_type = Text))]
+#[serde(rename_all = "lowercase")]
+pub enum PostEffectType {
+    Phosphor,
+    SSAO,
```

Comment:
Should be `Ssao` (https://rust-lang.github.io/api-guidelines/naming.html#casing-conforms-to-rfc-430-c-case)